web-app/src/auth/auth.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { parseGameStateFromFile } from '../common/parseConfigFile';
+import { join } from 'path';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import {
+    GameState,
+    updateGameState,
+    initNewGameState,
+    DEFAULT_BOARD_SIZE,
+} from '../common/gameState';
+import { Coordinates, deepCloneObject } from '../common/utils';
+import { User } from '../users/user.entity';
+
+
+@Injectable()
+export class GamesService {
+
+    constructor(
+        @InjectRepository(User)
+        private userRepository: Repository<User>,
+    ) { }
+
+    async checkCredentials({ login }): string {
+        const user = await this.userRepository.findOne({ where:
+            { username: login }
+        });
+        if (user === undefined) {
+            throw new Error('unknown user')
+        }
+        
```

Comment:
Your IDE seems to not be correctly configured with prettier or eslint, you should take 5 min to configure it correctly :)